src/coreclr/src/inc/palclr.h

```diff
@@ -606,10 +606,27 @@
 #define MAKEDLLNAME_W(name) name W(".dll")
 #define MAKEDLLNAME_A(name) name  ".dll"
 
+// Target platform-specific library naming
+//
+#ifdef TARGET_WINDOWS
+#define MAKE_TARGET_DLLNAME_W(name) name W(".dll")
```

Comment:
Would it make sense to put this into `src\coreclr\src\inc>notepad crosscomp.h` where other similar definitions are? It should avoid the duplication between palclr.h and pal.h.

Reply:
I was wondering where to put it.  Seems as good as any.